### DISCUSSION TITLE
Explanation Lime

### DISCUSSION THREAD

**Discussion by daryraa** [teamalgoritma/community#397] [Q&A] [answered]
ketika saya menggunakan explanation terdapat error sepertinya di bagaian n_featuresnya 
![image](https://user-images.githubusercontent.com/78478403/194961552-fa939fd8-b9d3-4cf7-9346-af2df9243c70.png)
sudah saya isi dengan angka yang lain namun errornya masih sama

**Comment by takdirzd** [comment, marked as answer]
n_features menunjukkan jumlah features/prediktor yang ada dalam data kakak, jadi perlu disesuaikan dengan data yg kakak gunakan. Masukkan angka sesuai dengan junlah prediktor yang ada

Semoga membantu..🙏

**Reply by daryraa** [reply]
oke baik kak terima kasih
